### PR TITLE
Rework translation page

### PR DESCRIPTION
**Pull request by MrGeneration**
This PR reworks the translation page and sheds up a little light on the new process.

Note:
If everything looks good for you *without* complaints, you can directly hit the `Merge` button.
The PR is landing in a pre-release branch and does not affect life documentation pages.

### CODE REVIEW

**Comment by MrGeneration** [comment]
Hey @mgruner it would be fantastic if you could have a look at this particular documentation part as well! 
Did I miss something? Something wrong? :)

**Review by mgruner — APPROVED** [review]
@MrGeneration I applied a few minor changes, please have a look if you agree.

**Comment by MrGeneration** [comment]
Apart from line length (fixed) lgtm.

File: system/translations.rst [hunk 1]
```diff
@@ -1,18 +1,91 @@
...
+------------------------------
+
+No problem at all! If you can't find the strings of your objects already within
+the translation list, please ensure to `add the translation strings`_ to your
+installation.
+
+Now you can translate your objects as needed.
+
+.. _add the translation strings:
+   https://docs.zammad.org/en/latest/admin/console/other-usefull-commands.html#add-translation
```
Comment (mgruner) [inline system/translations.rst:52]: @MrGeneration btw, it should be `useful` with just one `l`. Maybe you can fix that someday.

**Pull request merged**